.github/ISSUE_TEMPLATE/sdk_bug_report.yml

```diff
@@ -0,0 +1,108 @@
+name: Apify SDK bug report
+description: Report incorrect or unexpected behavior of a module
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        If you have a question, please ask it on [GitHub Discussions](https://github.com/apify/crawlee/discussions) or in our [Discord server](https://discord.gg/jyEM2PRvMU)
+
+  - type: dropdown
+    id: package
+    attributes:
+      label: Which package is this bug report for? If unsure which one to select, leave blank
+      multiple: false
+      options:
+        - "apify"
+        - "@apify/scraper-tools"
+    validations:
+      required: false
```

Comment:
lets comment this out for now, scraper-tools is internal package, we dont document it anywhere, we cant expect users to know what its about, and we dont have any other package in the monorepo